control-plane/stor-port/src/types/v0/store/registry.rs

```diff
@@ -29,6 +29,10 @@ impl CoreRegistryConfig {
     pub fn mayastor_compat_v1(&self) -> bool {
         self.mayastor_compat_v1.unwrap_or(false)
     }
+    /// Set the `mayastor_compat_v1` to true.
+    pub fn enable_mayastor_compat_v1(&mut self) {
```

Comment:
maybe set instead, so we can disable it when it's not needed anymore?

Reply:
We should disable it automatically when there are no more nexus info coming from the old location. We can check this in startup?

Reply:
How do we detect that? Its on root.

Reply:
We would probably need to go though all the keys and find out which one only uuid4 format. Seems to be a costly operation.

Reply:
As discussed we will take this as enhancement